### ISSUE TITLE
Do not hide PAP on connui-internet

### ISSUE THREAD

**Issue by MerlijnWajer** [maemo-leste/bugtracker#136]
Should be a no-brainer?

```23:23 < Wizzup> freemangordon: lol :)
23:23 < Wizzup> can we please unlock 'EAP PAP' for TTLS
23:23 < Wizzup> I think it is hidden behind some gconf key
23:24 < Wizzup> per https://bugs.maemo.org/show_bug.cgi?id=1635#c107
23:38 < Wizzup>   pap_enabled = gconf_client_get_bool(gconf,
23:38 < Wizzup>                                       ICD_GCONF_SETTINGS"/ui/pap_enabled",
23:38 < Wizzup>                                       NULL);```

**Comment by MerlijnWajer** [comment]
What I mean is, we should just remove the entire pap_enabled code/check. Always show it.

**Comment by sicelo** [comment]
+1

**Comment by MerlijnWajer** [comment]
Fixed in `connui-internet 2.78.8` and `connui-wlan 2.47.3`
